isso/views/comments.py

```diff
@@ -516,14 +517,13 @@ def delete(self, environ, request, id, key=None):
     @apiSuccessExample {html} Using GET:
         &lt;!DOCTYPE html&gt;
         &lt;html&gt;
-            &lt;head&gt;
-                &lt;script&gt;
-                    if (confirm('Delete: Are you sure?')) {
-                        xhr = new XMLHttpRequest;
-                        xhr.open('POST', window.location.href);
-                        xhr.send(null);
-                    }
-                &lt;/script&gt;
+        &lt;head&gt;
```

Comment:
This change appears unrelated, why is it necessary?

Reply:
I updated some docs on occasion.

Reply:
Please split that change out to a separate pull request.

Reply:
Undone. Feel free to reintroduce yourself, I’m not going to create a PR for this.

Reply:
Thanks; it's just that random unrelated changes make the pull request and subsequent merge commit harder to understand, especially for somebody not intimately familiar with all parts of isso's codebase, like myself.